aptos-move/block-executor/src/code_cache_global.rs

```diff
@@ -124,24 +126,22 @@ where
     /// Inserts modules into the cache.
     /// Notes:
     ///   1. Only verified modules are inserted.
-    ///   2. Valid modules should not be removed, and new modules should have unique ownership. If
-    ///      these constraints are violated, a panic error is returned.
-    pub fn insert_verified_unsync(
+    ///   2. Not overridden modules should not be removed, and new modules should have unique
```

Comment:
where do we check unique ownership?

Reply:
I think it's currently cloned for MVHashMap, so the comment will only be true once we remove the clone (there is a TODO)